projects/storefrontlib/src/cms-components/product/product-list/product-facet-navigation/product-facet-navigation.component.ts

```diff
@@ -99,12 +98,12 @@ export class ProductFacetNavigationComponent implements OnInit, OnDestroy {
     }
   }
 
-  getVisibleFacetValues(facet): any {
+  getVisibleFacetValues(facet: Facet): any {
```

Comment:
We can improve typesafety here by replacing `any` with `FacetValue[]` :)

Reply:
good, I have refactored this completely in a POC, and will bring this in a formal PR shortly. Hence i havent looked at the existing implementation for this PR. Either way, have done what you said ;)